ruby/vampires.rb

```diff
@@ -0,0 +1,131 @@
+#Interview with the Vampire
+
+require 'date' #import date method
+
+def survey_session(times) #Method to process nultiple employees - Release 3
+	counter = 0
+	while counter < times # Loop to call survey_form according to times
+		survey_form # Call the method survey_form
+	counter +=1
+	end
+	times
+end
+
+def survey_form
+	puts "What's your name?"
+	employee_name = gets.chomp
+
+	# To compare the answers to see if the informed_age matches the informed_birth:
+	puts "What will be your age at the last day of this year?"
+	informed_age = gets.chomp.to_i #convert input to integer
+
+	puts "What year were you born?"
+	informed_birth = gets.chomp.to_i 
+	age = Date.today.year - informed_birth #get current year 
+
+	puts "Our company cafeteria serves garlic bread. Should we order some for you? (y/n)"
+	wants_garlic = nil # 1.initialize variable to use on loop
+
+	# Create loop to check if input is valid
+	until wants_garlic == 'y' || wants_garlic == 'n'
+		wants_garlic = gets.chomp.downcase #use .downcase to make input "case insensitive"
+		if wants_garlic == 'y' 
```

Comment:
There is some refactoring that could be done, but the conditional logic and logic flow is good.  For instance (imagine with your own variable names): 
```
if garlic_bread.downcase == "yes" || garlic_bread.downcase == "y"
    garlic_bread = true
else
   garlic_bread = false
end
```
can be refactored to:
```
garlic_bread = garlic_bread.downcase == "yes" || garlic_bread.downcase == "y"
```
If `garlic_bread.downcase` is neither of those answers, `garlic_bread` will be set to false.